blended-measurement/blended-measurement.tex

```diff
@@ -0,0 +1,685 @@
+
+Transient events that affect only a small fraction of exposures but don't move
+are some what more difficult to handle.  Many of these will be
+straightforwardly detected in single-epoch difference images, and others will
+be detected in special coadds that cover only limited epochs.  However, we
+probably want to mask and reject these objects entirely when building coadds,
+so it will be impossible to deblend them there.  Instead, we'll have to add
+them back in when we transition to multi-epoch measurement (which should be
+straightforward, as their positions will be known, and we'll assume they're
+point sources).
+
+Fast moving solar system objects will similarly be detected in single-epoch
+difference images, and have orbits determined from these detections.  We'll
+also want to mask and reject them from coadds.  We still want to
+include them in multi-epoch measurement, both to ensure overlapping static
+objects are handled correctly and to measure flux as a function of time for
+the moving objects.  We'll use a trailed model for at these (which of course
+approaches a point source as the speed decreases).
+
+Extended variable or transient objects such as comets and supernova light
+echoes will be much harder to model or otherwise deblend in multi-epoch
+measurement, and our assumption for now is that these will be best analyzed
+via difference images, and hence in coaddition and multi-epoch measurement
+we'll simply mask them out.
+
+\section{LSST Pipeline Straw-Man Proposal}
+
+The above sections describe a number of algorithmic options that can be
+combined in myriad ways.  In this section, we describe (at summary level) a
+full baseline pipeline and a few top-priority alternatives. The baseline plan
+is outlined in Figure~\ref{fig:flowchart}, with details described in the next
+section and alternatives in Section~\ref{sec:alternatives}.
+
+\begin{figure}
+\includegraphics[width=\columnwidth]{flowchart}
+\caption{Baseline Pipeline for Blended Measurement}
+\label{fig:flowchart}
+\end{figure}
+
+\subsection{Baseline}
+
+The first major processing stage described here is the Deblender [P1], which
+we imagine as an algorithm very similar to that the SDSS {\em Photo} deblender
+described in Section~\ref{sec:deblender}, likely using a symmetry ansatz to
+define templates.  The inputs will be a detection catalog containing merged
+{\texttt Footprints} and {\texttt Peaks} from all detection images [D1], and
+at least one coadd image per filter [D2].  We may have multiple input coadds
+for each filter, representing different depth vs. resolution tradeoffs or
+different epochs, and possibly some coadds that represent combinations of data
+from different filters. The details of these inputs and the parallelization
+and data flow within the deblender itself are beyond the scope of this
+document. The outputs are deblended pixel values for both direct [D4] and PSF-
+matched [D5] coadds (generated by sequentially replacing neighbors with noise,
```

Comment:
I was trying to simplify the flowchart here by rolling noise replacement into the deblender, so the "deblended pixel values" I'm referring to here are actually the noise-replaced images (since producing the noise-replaced images is all we use the `HeavyFootprints` for).  If that was confusing, I should probably just make the noise replacement nodes explicit.